services/catalog/Dockerfile

```diff
@@ -117,19 +118,27 @@ WORKDIR /home/scu
 # Starting from clean base image, copies pre-installed virtualenv from prod-only-deps
 COPY --chown=scu:scu --from=prod-only-deps  ${VIRTUAL_ENV} ${VIRTUAL_ENV}
 
+# wget required by wait-for
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends \
+  wget \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/*
+
 # Copies booting scripts
 COPY --chown=scu:scu services/catalog/docker services/catalog/docker
 RUN chmod +x services/catalog/docker/*.sh
-
+COPY --chown=scu:scu scripts/common-docker-boot/wait-for /usr/local/bin/wait-for
+RUN chmod +x /usr/local/bin/wait-for
 
 HEALTHCHECK --interval=30s \
-    --timeout=20s \
-    --start-period=30s \
-    --retries=3 \
-    CMD ["python3", "services/catalog/docker/healthcheck.py", "http://localhost:8000/"]
+  --timeout=20s \
+  --start-period=30s \
+  --retries=3 \
+  CMD ["python3", "services/catalog/docker/healthcheck.py", "http://localhost:8000/"]
 
 ENTRYPOINT [ "/bin/sh", "services/catalog/docker/entrypoint.sh" ]
-CMD ["/bin/sh", "services/catalog/docker/boot.sh"]
+CMD ["/bin/sh", "-c",  "wait-for http://migration:8000 -- services/catalog/docker/boot.sh"]
```

Comment:
never too sure whether the -c is needed or not. and also please use the verbose option (for all the Dockerfiles)